README.md

```diff
@@ -126,11 +123,19 @@ The user wallet set in `.env` must have native tokens for each of the Subnets us
 
 ## Upgradeability
 
-The Teleporter contract is non-upgradeable and can not be changed once it is deployed. This provides immutability to the contracts, and ensures that the contract's behavior at each address is unchanging. However, to allow for new features and potential bug fixes, new versions of the Teleporter contract can be deployed to different addresses. The [TeleporterRegistry](./contracts/src/Teleporter/TeleporterRegistry.sol) is used to keep track of the deployed versions of Teleporter, and to provide a standard interface for dApps to interact with the different Teleporter versions.
+<<<<<<< HEAD
```

Comment:
Looks like there are some merge artifacts, and some duplicated paragraphs in this file

Reply:
Whoops, should be fixed now